packages/opentelemetry-plugin-http/test/functionals/http-enable.test.ts

```diff
@@ -24,15 +24,15 @@ import { HttpPlugin, plugin } from '../../src/http';
 import { assertSpan } from '../utils/assertSpan';
 import { DummyPropagation } from '../utils/DummyPropagation';
 import { httpRequest } from '../utils/httpRequest';
-import { TracerTest } from '../utils/TracerTest';
-import { SpanAuditProcessor } from '../utils/SpanAuditProcessor';
+import { TestProcessor } from '../utils/TestProcessor';
+import { NodeTracer } from '@opentelemetry/node-tracer';
 
 let server: http.Server;
 const serverPort = 12345;
 const protocol = 'http';
 const hostname = 'localhost';
 const pathname = '/test';
-const audit = new SpanAuditProcessor();
+const spanProcessor = new TestProcessor();
```

Comment:
Instead of `TestProcessor`, I'd suggest to use `SimpleSpanProcessor` and `InMemorySpanExporter` similar to gRPC plugin: https://github.com/open-telemetry/opentelemetry-js/pull/215/files#diff-629fcf95dfdc7b02a1d4f0742fb1cd17R513.